### DISCUSSION TITLE
VST Midi settings not saved

### DISCUSSION THREAD

**Discussion by dobo365** [RomanKubiak/ctrlr#598] [Q&A]
I noticed that the Midi settings are not kept when creating a new track or a new DAW project with a VST version of a panel.
It is each time needed to set the Midi settings (and sometimes other settings).
Settings are kept when saving the project and re-opening it BUT not when adding a new track, even in the same project.
Settings are always kept with the standalone version.

Anyone having some idea or is it something we will have to code?
I would have expected that the VST would pull the settings from the same temp settings file as the standalone.

BUT maybe that this has not been done on purpose as each track using the VST can get different settings...

**Comment by damiensellier** [comment]
Hi Dominique, I noticed that too, you have to setup your plugin options like window size, midi i/o etc and "save as default preset" in your host plugin menu. It's working great, this way you'll have all your options saved in the host preferences. It's working in cubase and studio one etc

![audio_effects_preset_management_pop_up_menu](https://github.com/RomanKubiak/ctrlr/assets/55164385/963cb84a-008e-4ef8-9088-c6dc22ef4279)

It stores in the host plugins default settings what you have for the Standalone in the Ctrlr.settings.
